charts/index.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+entries:
+  tekton-pipelines:
+  - apiVersion: v1
+    appVersion: "1.0"
+    created: "2019-12-13T16:30:34.228777-05:00"
+    description: A Helm chart for Tekton Pipelines
+    digest: 8f93704f1a1f3befdb454c50ed74d4466b7534f360382b73bc8708b920eb2c63
+    icon: https://raw.githubusercontent.com/jenkins-x/jenkins-x-platform/master/images/go.png
+    name: tekton-pipelines
+    urls:
+    - https://raw.githubusercontent.com/adjavaherian/pipeline/master/charts/tekton-pipelines/tekton-pipelines-0.9.0.tgz
```

Comment:
This is currently pointing at my personal fork for testing the Helm install.  Will change this url for the official Tekton release.